openedx/core/lib/api/utils.py

```diff
@@ -19,23 +19,39 @@ def pluggable_override(override):
         ...         return _something()
 
     2. Prepare an alternative implementation:
-        >>> def do_something_different(something):
-        ...     return getattr(something, '_different')()
-
+        >>> def do_something_different(prev_fn, something):
+        ...     result = getattr(something, '_different')()
+        ...     return result or prev_fn()
 
     3. Specify the path in settings (e.g. in `envs/private.py`):
         >>> DO_SOMETHING_IMPL = 'do_something_different_plugin.do_something.do_something_different'
+
+        You can also chain functions:
+        >>> DO_SOMETHING_IMPL = [
+        ...     'do_something_different_plugin.do_something.do_something_different',
+        ...     'do_something_else_plugin.do_something.do_something_else',
+        ... ]
     """
     def decorator(f):
         @functools.wraps(f)
         def wrapper(*args, **kwargs):
-            function_string = getattr(settings, override)
-            if not function_string:
-                return f(*args, **kwargs)
-
-            module, function = function_string.rsplit('.', 1)
-            mod = import_module(module)
-            func = getattr(mod, function)
-            return func(*args, **kwargs)
+            prev_fn = f  # The base method in `edx-platform`.
```

Comment:
This might need to be `partial(f, *args, **kwargs)` because it becomes the `prev_fn` arg later, and the other prev_fn args already have the parameters applied?

Reply:
That's a good point. Thank you for catching this!